pylintfileheadertest/file_header_checker_test.py

```diff
@@ -43,15 +43,6 @@ def test_valid_header_not_at_top_message_added(self):
                 args='# Valid\n# Header')):
             self.checker.process_module(node_mock)
 
-    def test_config_empty_no_message_added(self):
```

Comment:
Please keep this test

Reply:
it's move to the different place

Reply:
see the `TestFileHeaderCheckerNoConfig` class

Reply:
https://github.com/HaaLeo/pylint-file-header/pull/3/files#diff-e07cba7b58fc9200116db4af58244482bbb5135e9026ff212871e130d37f198fR71